index.html

```diff
@@ -230,6 +230,6 @@ <h4 class="popup__type">Квартира</h4>
       <button type="button" class="error__button">Попробовать снова</button>
     </div>
   </template>
-
+  <script src="/js/main.js"></script>
```

Comment:
```suggestion
  <script src="./js/main.js"></script>
```
лучше делай относительные импорты